src/class3_Queue_Stack_LinkedList/CheckIfLinkedListHasACycle.java

```diff
@@ -0,0 +1,21 @@
+package class3_Queue_Stack_LinkedList;
+
+/**
+ *
+ Description
+ Check if a given linked list has a cycle. Return true if it does, otherwise return false.
+ */
+
+public class CheckIfLinkedListHasACycle {
+    public boolean hasCycle(ListNode head) {
+        if (head == null) return false;
+        ListNode fast = head;
```

Comment:
`fast` should initially faster than `slow`, otherwise, if `head.next == null` while loop is checking fast but not slow

Reply:
done